{"timestamp":1729470430545,"message":"Error parsing DOAB record rec2","log.level":"ERROR","logger.name":"processes.doab","thread.id":140073765616512,"thread.name":"MainThread","process.id":2411,"process.name":"MainProcess","file.name":"/home/runner/work/drb-etl-pipeline/drb-etl-pipeline/processes/doab.py","line.number":117,"entity.type":"SERVICE"}
